### DISCUSSION TITLE
Wallpaper swtich not working

### DISCUSSION THREAD

**Discussion by mrhalliwell** [AutoDarkMode/Windows-Auto-Night-Mode#843] [Wallpaper]
I wasn't sure what is wrong here. I have set up a light theme with its wall paper and a dark theme with another wallpaper. Enable wallpaper swtich is checked. But when the predetermined time comes, the wallpaper switched for one second or so and then switched back to light theme wallpaper. I am not sure what to do here. Can anyone help?
BTW, love this app. Thanks for this!

**Comment by Armin2208** [comment]
Fixed in Beta release, will be fixed in a future release. Please be patient.
